cmd/zdb/zdb.c

```diff
@@ -2483,6 +2499,9 @@ dump_znode(objset_t *os, uint64_t object, void *data, size_t size)
 		}
 		(void) printf("\tpath	%s\n", path);
 	}
+
+	if S_ISLNK(mode)
```

Comment:
There needs to be parenthesis around the `if` here.

Reply:
Ouch:) Thanks. Fixed.